crates/core/tedge_api/src/messages.rs

```diff
@@ -595,11 +596,19 @@ impl LogUploadCmdPayload {
         }
     }
 
-    pub fn add_reason(self, reason: &str) -> Self {
-        Self {
-            reason: Some(reason.into()),
-            ..self
-        }
+    pub fn executing(&mut self) -> () {
```

Comment:
```suggestion
    pub fn executing(&mut self) {
```
unneeded unit return

Reply:
fixed